common/org.testeditor.dsl.common/src/org/testeditor/dsl/common/util/OutputStreamCopyUtil.xtend

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2016 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+package org.testeditor.dsl.common.util
+
+import java.io.BufferedReader
+import java.io.IOException
+import java.io.InputStream
+import java.io.InputStreamReader
+import java.io.PrintStream
+import org.slf4j.LoggerFactory
+
+class OutputStreamCopyUtil extends Thread {
```

Comment:
We use it also in MavenExecutor. It was first a inner class of the MavenExecutor. I extracted it to reuse it in GradleCommand.